st2common/st2common/models/utils/action_param_utils.py

```diff
@@ -79,7 +79,7 @@ def is_immutable(param_meta):
     return (required_params, optional_params, immutable_params)
 
 
-def cast_params(action_ref, params):
+def cast_params(action_ref, params, cast_overrides=None):
```

Comment:
It would be good to add a docstring which describes what `cast_overrides` parameter does.